app/config/locale/translations/gu.countries.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+return [
+
+    'AF' => 'અફઘાનિસ્તાન',
+    'AO' => 'અંગોલા',
+    'AL' => 'અલ્બેનિયા',
+    'AD' => 'એન્ડોરા',
+    'AE' => 'સંયુક્ત આરબ અમીરાત',
+    'AR' => 'આર્જેન્ટિના',
+    'AM' => 'આર્મેનિયા',
+    'AG' => 'એન્ટિગુઆ અને બાર્બુડા',
+    'AU' => 'સ્ટ્રેલિયા',
+    'AT' => 'ઑસ્ટ્રિયા',
+    'AZ' => 'અઝરબૈજાન',
+
+    'BI' => 'બરુંડી',
+    'BE' => 'બેલ્જિયમ',
+    'BJ' => 'બેનિન',
+    'BF' => 'બુર્કિના ફાસો',
+    'BD' => 'બાંગ્લાદેશ',
+    'BG' => 'બલ્ગેરિયા',
+    'BH' => 'બહરીન',
+    'BS' => 'બહામાસ',
+    'BA' => 'બોસ્નિયા અને હર્ઝેગોવિના',
+    'BY' => 'બેલારુસ',
+    'BZ' => 'બેલીઝ',
+    'BO' => 'બોલિવિયા',
+    'BR' => 'બ્રાઝિલ',
+    'BB' => 'બાર્બાડોઝ',
+    'BN' => 'બ્રુનેઇ',
+    'BT' => 'ભૂટાન',
+    'BW' => 'બોત્સ્વાના',
+    'CF' => 'સેન્ટ્રલ આફ્રિકન રિપબ્લિક',
+    'CA' => 'કેનેડા',
+    'CH' => 'સ્વિટ્ઝર્લન્ડ',
+    'CL' => 'ચિલી',
+    'CN' => 'ચાઇના',
+    'CI' => 'આઇવરી કોસ્ટ',
+    'CM' => 'કેમરૂન',
+    'CD' => 'ડીઆર કોંગો',
+    'CG' => 'રિપબ્લિક ઓફ કોંગો',
+    'CO' => 'કોલમ્બિયા',
+    'KM' => 'કોમોરોઝ',
+    'CV' => 'કેપ વર્ડે',
+    'CR' => 'કોસ્ટા રિકા',
+    'CU' => 'ક્યુબા',
+    'CY' => 'સાયપ્રસ',
+    'CZ' => 'ચેકિયા',
+    'DE' => 'જર્મની',
+    'DJ' => 'જીબુટી',
+    'DM' => 'ડોમિનિકા',
+    'DK' => 'ડેનમાર્ક',
+    'DO' => 'ડોમિનિકન રિપબ્લિક',
+    'DZ' => 'અલ્જેરિયા',
+    'ER' => 'એરિટ્રીઆ',
+    'EC' => 'એક્વાડોર',
+    'EG' => 'ઇજિપ્ત',
+    'IR' => 'એરિટ્રીઆ',
+    'ES' => 'સ્પેન',
+    'EE' => 'એસ્ટોનીયા',
+    'ET' => 'ઇથોપિયા',
+    'FI' => 'ફિનલેન્ડ',
+    'FJ' => 'ફીજી',
+    'FR' => 'ફ્રાંસ',
+    'FM' => 'માઇક્રોનેસીયા',
+    'GA' => 'ગેબોન',
+    'GB' => 'યુનાઇટેડ કિંગડમ',
+    'GE' => 'જ્યોર્જિયા',
+    'GH' => 'ઘાના',
+    'GN' => 'ગિની',
+    'GM' => 'ગાંબિયા',
+    'GW' => 'ગિની-બિસાઉ',
+    'GQ' => 'ઇક્વેટોરિયલ ગિની',
+    'GR' => 'ગ્રીસ',
+    'GD' => 'ગ્રેનાડા',
+    'GT' => 'ગ્વાટેમાલા',
+    'GY' => 'ગુયાના',
+    'HN' => 'હોન્ડુરાસ',
+    'HR' => 'ક્રોએશિયા',
+    'HT' => 'હૈતી',
+    'HU' => 'હંગેરી',
+    'ID' => 'ઇન્ડોનેશિયા',
+    'IN' => 'ભારત',
+    'IE' => 'આયર્લેન્ડ',
+    'IR' => 'ઈરાન',
+    'IQ' => 'ઇરાક',
+    'IS' => 'આઇસલેન્ડ',
+    'IL' => 'ઇઝરાઇલ',
+    'IT' => 'ઇટાલી',
+    'JM' => 'જમૈકા',
+    'JO' => 'જોર્ડન',
+    'JP' => 'જાપાન',
+    'KZ' => 'કઝાકિસ્તાન',
+    'KE' => 'કેન્યા',
+    'KG' => 'કિર્ગીસ્તાન',
+    'KH' => 'કંબોડિયા',
+    'KI' => 'કિરીબતી',
+    'KN' => 'સેન્ટ કિટ્સ એન્ડ નેવિસ',
+    'KR' => 'દક્ષિણ કોરિયા',
+    'KW' => 'કુવૈત',
+
```

Comment:
Please remove the extra space

Reply:
@eldadfux That's come by default in Gujrati I have tried to remove the spaces but it changes the actual meaning of the word.

Reply:
Done @eldadfux